shell/components/form/ResourceTabs/index.vue

```diff
@@ -233,6 +237,7 @@ export default {
     v-bind="$attrs"
     :default-tab="defaultTab"
     :resource="value"
+    :use-hash="useHash"
```

Comment:
We need to be able to turn off using the hash associated with tabs used on edit pages that we display in the ResourceDetailDrawer.

When a tab is rendered or selected it updates the hash in the url. Url changes trigger the slideIn/Drawer to close. Since we shouldn't need the url to reference a tab in the drawer I think it's safe to turn it off in these circumstances.